dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -235,7 +248,7 @@ private static ArtifactResults generateArtifactReport(
       Artifact artifact,
       ArtifactInfo artifactInfo,
       List<DependencyGraph> globalDependencies,
-      ImmutableMap<Path, JarLinkageReport> jarToLinkageReport,
+      Collection<JarLinkageReport> staticLinkageCheckReports,
```

Comment:
This is really any collection, not something more specific like a list or a set?

Reply:
Now it's `Set`, after changing `ImmutableMultimap` to more specific `ImmutableSetMultimap` in the caller.